gnd/src/main/java/com/google/android/gnd/ui/home/mapcontainer/MapContainerViewModel.java

```diff
@@ -169,12 +170,13 @@
                 ImmutableSet.of(
                     toMapPolygon(
                         featureRepository.newPolygonFeature(
-                            selectedProject.get(), selectedLayer.get(), vertices))));
-    this.mapFeatures =
-        LiveDataReactiveStreams.fromPublisher(
-            persistentFeatures.withLatestFrom(
-                transientFeatures.startWith(ImmutableSet.<MapFeature>of()),
-                MapContainerViewModel::concatFeatureSets));
+                            selectedProject.get(), selectedLayer.get(), vertices))))
+            .startWith(ImmutableSet.<MapFeature>of());
```

Comment:
`startWith` is added below so it's not needed here.

Reply:
removed startWith